DataBiosphere/Saturn-UI
# [AN-229] Add batch workspace setting

### Jira Ticket: https://broadworkbench.atlassian.net/browse/AN-228




## Summary of changes:


### What
- Add GCP Batch workspace setting option under a closed feature flag
- Only users added to the 'batch-preview' group will be able to enable the setting
- If enabled, cromwell will run workflows on GCP Batch

<img width="609" alt="image" src="https://github.com/user-attachments/assets/93b536c9-8a70-485e-86b9-02de5e43f10b" />
<img width="1420" alt="image" src="https://github.com/user-attachments/assets/29f53987-ca2b-462d-acfb-b378d4f4b1a0" />

### Dependencies
Workspace setting name from https://github.com/broadinstitute/firecloud-orchestration/pull/1503/files

### Testing strategy

- Create private group 'batch-preview'
- Enable the batch setting in a workspace
- Check other user not in the group cannot enable the batch setting
- Run a workflow, check that it ran on GCP batch
Proof of success
![image](https://github.com/user-attachments/assets/3a050b1d-3790-4390-8a02-e3aa57588670)
